Formula/g/go@1.19.rb

```diff
@@ -23,6 +18,10 @@ class GoAT119 < Formula
 
   keg_only :versioned_formula
 
+  # EOL with Go 1.21 release (2023-08-08)
+  # Ref: https://go.dev/doc/devel/release#policy
+  deprecate! date: "2023-08-13", because: :unsupported
```

Comment:
if it's EOL, maybe just disable it in a year?

Reply:
you mean change it to disable?

it does have sizable traffic to it, https://formulae.brew.sh/formula/go@1.19 (I would prefer deprecate now vs disable now :) )